components/kyma-environment-broker/internal/orchestration/handlers/kyma_handler_test.go

```diff
@@ -61,6 +61,45 @@ func TestHandler_AttachRoutes(t *testing.T) {
 	})
 }
 
+// Testing Kyma Version is disabled due to GitHub API RATE limits
+func TestHandler_KymaVersion(t *testing.T) {
+	t.Run("kyma version validation", func(t *testing.T) {
```

Comment:
You could implement a fake github client returning the response you wish to test